package.json

```diff
@@ -1,36 +1,70 @@
 {
-  "name": "winston-logstash",
-  "version": "0.4.0",
+  "name": "@dkuida/winston-logstash",
+  "version": "0.5.1",
   "description": "A Logstash transport for winston",
-  "main": "./lib/winston-logstash",
-  "homepage": "https://github.com/jaakkos/winston-logstash",
+  "main": "dist/winston-logstash",
+  "homepage": "https://github.com/dkuida/winston-logstash",
   "scripts": {
-    "test": "node_modules/mocha/bin/mocha test/*_test.js"
+    "test": "mocha --require ts-node/register test/*_test.ts",
+    "build": "npm run clean && npm run tsc && npm run copy-types",
+    "copy-types": "copyfiles -u 1 ./src/*.d.ts ./dist",
+    "lint": "tslint -p tsconfig.json",
+    "lint:fix": "tslint -p tsconfig.json --fix",
+    "clean": "rimraf dist/*",
+    "tsc": "tsc --declaration",
+    "ts:watch": "tsc --watch",
+    "deploy": "npm run build && npm version patch -m \"Release v%s\" && npm publish",
+    "preversion": "npm run test",
+    "postversion": "git push && git push --follow-tags"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=8"
+  },
+  "publishConfig": {
+    "access": "public"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/jaakkos/winston-logstash.git"
+    "url": "git+https://github.com/dkuida/winston-logstash.git"
```

Comment:
Probably should go back to previous value

Reply:
you are right - needed to use it asap - and wouldn't create a package with someone else name. removed